devops/chocolatey-test-environment.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>chocolatey-test-environment</id>
+    <version>0.10.15</version>
```

Comment:
I would not base this off of Chocolatey's version - I would let it have its own version so it can be released independently.